### PR TITLE
Create Rule "change-date-of-existing-commit/rule"

### PR DESCRIPTION
**Pull request by jeoffreyfischer**
>
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

✏️ A conversation with Igor and an email task to myself

> 2. What was changed?

✏️ A new rule was created to change the date of an existing commit

> 3. Did you do pair or mob programming (list names)?

✏️ Igor

### CODE REVIEW

File: rules/change-date-of-existing-commit/rule.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+---
+
+Updating commit information can be essential for maintaining accurate project history or correcting errors.
+Whether you need to change a commit date for clarity, compliance, or other reasons, you have a couple of methods at your disposal.
+This rule outlines how to change the date of an existing commit using both a manual CLI approach and an automated script.
+
+<!--endintro-->
+
+# Method 1 – Use CLI
```
Comment (tiagov8) [inline rules/change-date-of-existing-commit/rule.md:24]: Hey @jeoffreyfischer 

These should be H2 (the main title is already H1)

- [ ] 1. Please fix

File: rules/change-date-of-existing-commit/rule.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+1. Checkout to the rule branch
```
Comment (tiagov8) [inline rules/change-date-of-existing-commit/rule.md:26]: Hey @jeoffreyfischer 

Is this the rule branch, OR any branch?

File: rules/change-date-of-existing-commit/rule.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+::: greybox
```
Comment (tiagov8) [inline rules/change-date-of-existing-commit/rule.md:27]: Hey @jeoffreyfischer 

- [ ] 1. Can you change these greyboxes to a code box?

e.g.

```
git checkout -b {{ BRANCH NAME }} origin/{{ BRANCH NAME }}
```

PS. Don't forget to add the language

**Review by tiagov8 — CHANGES_REQUESTED** [review]
Hey @jeoffreyfischer 

This new rule needs to be added to a category so it's not orphaned

- [ ] 1. Please action

File: rules/change-date-of-existing-commit/rule.md (hunk 1 above)
Comment (jeoffreyfischer) [reply rules/change-date-of-existing-commit/rule.md]: Done

File: rules/change-date-of-existing-commit/rule.md (hunk 3 above)
Comment (jeoffreyfischer) [reply rules/change-date-of-existing-commit/rule.md]: Done - I used: ``` bash

**Review by tiagov8 — APPROVED** [review]

**Pull request merged**